File: gateOne/day 3/java/Task1.java
import java.util.Random;
import java.util.Scanner;
public class Task1{
	public static void main(String... args){
	Random secureRandom = new Random();
	Scanner input = new Scanner(System.in);

	int correctAnswers = 0;
	int inCorrectAnswers = 0;
	for(int count = 0; count < 10; count++){
		int firstNumber = secureRandom.nextInt(-10, 10);
		int secondNumber = secureRandom.nextInt(-10, 10);
		String operand = "+";
		int answer = 0;
		if(operand == "+"){
			answer = firstNumber + secondNumber;
		}

		
				
		System.out.print(firstNumber + " " + operand + " " + secondNumber + "= ");
		int userAnswer = input.nextInt();

		if(answer == userAnswer){
			correctAnswers += 1;
		}else{
			inCorrectAnswers += 1;		
		}
		System.out.println();
	}

	System.out.println("the number of correct answers is " + correctAnswers);
	System.out.print("the number of incorrect answers is " + inCorrectAnswers);		


	}


}

File: gateOne/day 3/python/task1.py
import random

correct_number = 0
incorrect_number = 0
for index in range(10):
	first_numbers = random.randrange(-10, 10)
	second_numbers = random.randrange(-10, 10)
	
	operand = random.choice(["+","-","/","*"])
	
	if operand == "+":
		answer = first_numbers + second_numbers
	elif operand == "-":
		answer = first_numbers - second_numbers
	elif operand == "/":
		answer = first_numbers / second_numbers
	elif operand == "*":
		answer = first_numbers * second_numbers
	
	
	answer = first_numbers + second_numbers	
	print(first_numbers," ",operand," ",second_numbers, " = ", end= "")
	user_input = int(input())
	if answer == user_input:
		correct_number += 1
	else:
		incorrect_number += 1
	
print("the number of correct answers is ", correct_number)
print("the number of incorrect answers is ", incorrect_number)